jenkins/catapult/jenkins/catapult-client-build-catapult-project-daily.groovy

```diff
@@ -73,6 +73,13 @@ pipeline {
 						}
 					}
 				}
+				stage('code coverage') {
+					steps {
+						script {
+							dispatch_build_job('gcc-10-code-coverage', 'tests-diagnostics')
```

Comment:
why tests-diagnostics? would probably not use that for cc build

Reply:
no specific reason.  will change to ``tests-metal``